actors/actors.go

```diff
@@ -0,0 +1,82 @@
+package actors
```

Comment:
I think this should be in the `builtin` package, since that's definitely what it's defining. That might prompt a rename of `BuiltinActor` to avoid stutter: maybe `Exported` or something?

Reply:
Almost all actors (account actor as an easy example) import the `builtin` package (since their addresses and codeIDs are defined there) leading to a circular dependency, which is why I put it here -- although I agree this is probably not the spot either. We could break the circular decency by:
- moving actor addresses and codes to a child package of builtin.
- move this file to a child package of builtin.
- something else?

Whatever solution we decide on I will land first and rebase this onto it.

Reply:
Ok. The builtin package is referenced fairly extensively in the impls, so moving the codes/addresses there would be a bit of merge pain. Which suggests option 2. 

Note we might move things again in attempting to [unify the lists](https://github.com/filecoin-project/specs-actors/pull/405#pullrequestreview-422322432).